### PR TITLE
Support installing plugins through deep linking

### PR DESCRIPTION
**Pull request by DesignCodeCreate**
Support for installing plugins from marketplace through deeplinking.

### CODE REVIEW

File: src-tauri/capabilities/main.json
Comment (ninjadev64) [inline src-tauri/capabilities/main.json:1]: This file should be indented with tabs

File: src/components/PluginManager.svelte [hunk 1]
```diff
@@ -121,6 +122,11 @@
 	(async () => plugins = await (await fetch("https://ninjadev64.github.io/openaction-plugins/catalogue.json")).json())();
 
 	let search: string = "";
+
+	onOpenUrl((urls: any) => {
+		let id = urls[0].split("installPlugin/")[1];
+		installPluginGitHub(id, plugins[id]);
+	});
```
Comment (ninjadev64) [inline src/components/PluginManager.svelte:129]: Please could you add checks for when the string does not contain installPlugin or the plugin ID is not present in the plugins object, and return doing nothing when this is the case

**Review by ninjadev64 — CHANGES_REQUESTED** [review]

**Comment by DesignCodeCreate** [comment]
Has been resolved

**Comment by ninjadev64** [comment]
LGTM, merged :+1:

**Pull request merged**